config.default.yaml

```diff
@@ -288,13 +288,15 @@ renewable:
       flowspeed: 1.0  # m/s
       # weight_with_height: false
       # show_progress: true
-      normalization: hydro_capacities # 'hydro_capacities' to normalize country production by using hydro_capacities, 'eia' to normalize by eia data, false for no normalization
     carriers: [ror, PHS, hydro]
     PHS_max_hours: 6
     hydro_max_hours: "energy_capacity_totals_by_country"  # not active
     clip_min_inflow: 1.0
     extendable: true
-    normalization_multiplier: 1.1  # multiplier applied after the normalization of the hydro production
+    normalization:
+      method: hydro_capacities  # 'hydro_capacities' to normalize country production by using hydro_capacities, 'eia' to normalize by eia data, false for no normalization
+      year: 2020  # year of statistic used to normalize the time series
```

Comment:
That makes the normalization way more complex and restrict the number of weather years and statistics that are possible.
I had to add this option also because of that.
When not provided, we could use the one from the weather year though.

Reply:
Agree that it's better to keep it simple. Using weather year can be a good solution if it does not make things too complex

Actually, `normalization_multiplier` is available in the config and could be used to account for multi-annual average for advanced analysis. So, it could be alright as it is but perhaps we'd need to highlight it in documentation :)